modules/snmp/metadata.yaml

```diff
@@ -8,7 +8,7 @@ modules:
         link: ""
         icon_filename: snmp.png
         categories:
-          - data-collection.snmp-and-networked-devices
```

Comment:
I think having SNMP in a separate category (snmp/network devices) made more sense but ok

Reply:
Having just a single integration in the category did not make sense when it can anyways be searched.. 
We may reintroduce this category based on user feedback.

cc: @ralphm @Ferroin

Reply:
i wish I was a user and can give feedback 😢

Reply:
I am creating a follow up issue for improving integrations with all of our feedback - as users of Netdata.. 👍